### PR TITLE
Feature/ui components

### PR DESCRIPTION
**Pull request by JessBrownless**

### CODE REVIEW

File: src/components/CTABlock/cta-block.tsx [hunk 1]
```diff
@@ -2,19 +2,18 @@ import React from "react"
...
-        </a>
+    <div className="bg-gray-100">
+      <div className="container mx-auto text-center pt-24 pb-24 w-9/12 md:w-8/12 lg:w-5/12">
+        <div className="pl-4 pr-4">
+          <h2 className="text-4xl mb-2">Book Your Appointment Today</h2>
+          <p className="mt-6 mx-auto mb-4">
+            Book your appointment with Dr Yalda today. A bespoke consultation,
+            without obligation, is provided before each treatment.
+          </p>
+          <a className="btn__primary inline-block mt-8" href="#">
```
Comment (sonnybooth-hexsu) [inline src/components/CTABlock/cta-block.tsx]: Can add link here as page in now available '/contact'

File: src/components/Filter/filter.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```
Comment (sonnybooth-hexsu) [inline src/components/Filter/filter.tsx]: Remove this please. Not needed now

File: src/components/Header/header.module.css [hunk 3]
```diff
@@ -49,11 +50,19 @@
...
+  font-size: 1rem;
   color: colorBlack;
-  text-decoration: underline;
+  text-decoration: none;
+  font-weight: medium;
+  font-weight: 500;
+  letter-spacing: 1;
 }
 
+/* TBC
```
Comment (sonnybooth-hexsu) [inline src/components/Header/header.module.css]: DELETE

File: src/components/TestimonialCard/testimonial-card.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ export const TestimonialCard = ({
   return (
     <div className="bg-white pt-16 pr-16 pl-16 pb-10 md:pl-24 shadow-md h-full">
       <div className="flex">
-        <div className="rounded-full h-12 w-12 mr-2 mb-4 overflow-hidden">
+        <div className="rounded-full h-12 w-12 mr-2 mb-4 overflow-hidden h-16 w-16">
```
Comment (sonnybooth-hexsu) [inline src/components/TestimonialCard/testimonial-card.tsx]: Remove old classes you have h-12 w-12 and h-16 w-16 all on one element

File: src/pages/contact.js [hunk 5]
```diff
@@ -2,114 +2,138 @@ import React from "react"
...
-              />
-              <p className="text-gray-600 text-xs italic">
-                Feel free to ask as many questions as you would like
+        </div>
+      </div>
+      <div className="container mx-auto pl-6 pr-6 pb-10 pt-20 pb-20">
+        <div className="pl-4 pr-4">
+          <div className="md:flex md:flex-wrap lg:flex-col lg:relative">
+            <div className="md:order-1 md:w-1/2">
+              <p className="text-center text-2xl pb-2 md:text-left md:pr-10 font-semibold">
```
Comment (sonnybooth-hexsu) [inline src/pages/contact.js]: Put media query classes in order. Move font-semibold before the md:'s

File: src/pages/contact.js [hunk 6]
```diff
@@ -2,114 +2,138 @@ import React from "react"
...
+                      className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200  py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+                      id="first-name"
+                      name="firstName"
+                      type="text"
+                      placeholder="Please enter your name"
+                    />
+                  </div>
+                  <div className="w-full md:w-1/2 px-3 lg:w-full">
+                    <label
+                      className="block uppercase tracking-wide text-gray-700 text-xs font-bold 1 mb-2"
```
Comment (sonnybooth-hexsu) [inline src/pages/contact.js]: Random number 1 here

File: src/pages/contact.js [hunk 7]
```diff
@@ -2,114 +2,138 @@ import React from "react"
...
+                <div className="flex flex-wrap -mx-3 mb-6">
+                  <div className="w-full px-3">
+                    <label
+                      className="block uppercase tracking-wide text-gray-700 text-xs font-bold 1 mb-2"
+                      htmlFor="message"
+                    >
+                      Message *
+                    </label>
+                    <textarea
+                      className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200  py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500 h-40"
```
Comment (sonnybooth-hexsu) [inline src/pages/contact.js]: Do focus ones after default styles. Just move h-40 to before the focus stuff here

File: src/pages/contact.js [hunk 8]
```diff
@@ -2,114 +2,138 @@ import React from "react"
...
+                      id="message"
+                      name="message"
+                      placeholder="Please leave your message here, be as brief or detailed as you like."
+                    />
+                  </div>
+                </div>
+                <button
+                  className="flex-shrink-0 bg-teal-500 hover:bg-teal-700 border-teal-500 hover:border-teal-700 text-sm border-4 text-white py-1 px-2  w-full px-4 py-4"
```
Comment (sonnybooth-hexsu) [inline src/pages/contact.js]: Again do all hover and focus and media query stuff after everything else

File: src/styles/global.css [hunk 9]
```diff
@@ -20,27 +20,88 @@ body {
   font-size: 17.6px;
 }
 
+p {
+  letter-spacing: 1;
+  /* color: #434181; */
```
Comment (sonnybooth-hexsu) [inline src/styles/global.css]: DELETE

File: src/styles/global.css [hunk 10]
```diff
@@ -20,27 +20,88 @@ body {
   font-size: 17.6px;
```
Comment (sonnybooth-hexsu) [inline src/styles/global.css:20]: Wtf is this value?

Comment (sonnybooth-hexsu) [reply src/styles/global.css:20]: Need to be in rem as well

File: src/styles/global.css [hunk 11]
```diff
@@ -20,27 +20,88 @@ body {
...
+  font-weight: 400;
+  text-align: center;
+
+  &:hover {
+    background-color: #7271A6;
+    transition: color .30s ease-in-out,background-color .30s ease-in-out,border-color .30s ease-in-out,box-shadow .30s ease-in-out; 
+  }
+}
+
+.btn__secondary {
```
Comment (sonnybooth-hexsu) [inline src/styles/global.css:61]: I'll sort all these

File: src/styles/global.css (hunk 10 above)
Comment (JessBrownless) [reply src/styles/global.css:20]: 17.6px. Why does it need to be in REM. I disagree

File: src/styles/global.css (hunk 11 above)
Comment (JessBrownless) [reply src/styles/global.css:61]: Thanks x

**Pull request merged**